src/job-details/JobDetails.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import {
+  Button,
+  Card, CardActions, CardContent, Typography,
+} from '@material-ui/core';
+import { ArrowDropDown } from '@material-ui/icons';
+import { Rating } from '@material-ui/lab';
+import { GET_JOB } from '../sources';
+import './styles.css';
+import CompanyDetails from './CompanyDetails';
+
+const JobDetails = () => {
+  const { id } = useParams();
+  const [companyCardVisibility, setCompanyCardVisibility] = useState(false);
+  const {
+    data, error,
+  } = useQuery(GET_JOB, { variables: { id: Number(id) } });
```

Comment:
This works as expected, just something to keep in mind : `The ID type is serialized in the same way as a String; however, it is not intended to be human‐readable. While it is often numeric, it should always serialize as a String.`